examples/compiler/01_lowering.ipynb

```diff
@@ -0,0 +1,363 @@
Pp5cXzL47Et1z5W45ZFDHudI4FYWdvzw/Lfsbd3TNHe9169XF2diE5OYnoI4dY/N1Cdu82smb9b5bfY0Hl9564Jisri++XLKLNgx156+2pdGzbmK2bf2H1qqV06doDk8nEuDdGANB/wNOENGhMZmbBVwW7Uf9hy39h755dTBj3aqGOSUTkXqWCRkRKvRmffJbr23KAAYOetTzu9sgD/L5rJ08PfZmnnx1205jxp06y9qeVpKYmExhUk85dHsXWNvt/uakpyezcsQ2/8hVxcXFl3U8rsC9bli5dH6Vipcps+GUtiYmnAdi924i7hyfB9zWgWfNWOP//h9z09PNs37oRH59y+Fepyo9LF2fn+e/HKF+hEiuXf8/x4zHUCw6hfYfs+yE4OTlbYsTGHudAHrNPHp5eNG/xgOW5ced2dv9u5PLlS1SrVp0Onbrg4uJaoBwLMw7lyvmxYvkPZJpMPNy5O4FBNfMdWxuDDU2bZS+1en2RcP3vcevmXxg84DEOH4piweezeWn4mJv+zq6X33viml+3beJUXCxvvT2FmrXq8PSzw5gzawYTx4+mXfuH+fp/X3Do4F94efnw2hvvFqrvm/XftNn9XLl8udAxRUTuVSpoRKTUSzuXSkry2RxtHp5eea5aUxCrVy3llWFPk55+3tIWfF8D/vftCvz8KnD4cBSDBzxGQPUg4k7GcvXqFQBmTH+PRd+s4OUXBpOcnATAzI+n8Ou2TUz/aD6DBzxmOZ0r7mQsgwc8Ro2atTl//rzl2otZn06jRcvWrFoZZul72IixvP7me5Z9atepx+AhLzJ21Au5cm/WvBU/rtqE2Wzm+Wf6sWL593h5+WBnZ0dCQjzVqlVn9bpfC5RjQcehTt37OBl7nPPn0yzjsGb9b9SqXfeWxh+gdZv29HliEF9+Pot161YWuqC52Xti8XcLcXP34KGHuwEwcvR4loZ9R9zJE0wcP4rlP34PwJtvfYCHZ+6LlW+3fxER+ZvWkRORUq/XYw9Rr1b5HD9JSYm3FCspKZFXhz+LyWTi2yVr+OtwIs8+N5z9kXt5642RObaNOXqEKR/OJurIGUaNncC51BTGvTGC3ZHHaNf+YQDmffEtS5dvyLe/w4eieGPce2zf+RfVAgJJSkrk+PGjbN/5F9M/mgfAku/+l2u/zl1DWRu+k7XhO5k2Y65lWdG27R8CYPfvRjZtXE+Dhk3Ys/8EuyOP07BRU44dO0qE8deb5liYcfjrQCTTZsxh196j3N+qDZcuXWT5ssWFGfY8hTTIvrdF3MnYQu97o/dEevp51qxaRuijvShb1gHIvmZn4jtTAfjffz8j7VwqjRo3o88Tg24p96J8T4qI3Os0QyMipV6bth3x8Mj5Lfq1D6qF9Uv4WtLSzvHv0F60adsRgLFvvMOXn8/ipzU/cuXK36cK1a1Xn959BgAw/JXX+fKzWfyx93cupKdbVuiyt7PH7gbLgvpXrsrjvfoD0PL+f3EsJpruj/YmoHoQ/pWrMuqVoSScPpVrP29vX7y9fYmNPc60yRPJysqib7/BDH/ldQAaN2lOVHQScSdPsHXLL0Qd2E909GEArmZcxd6+7A1zLMw4VA+swb9DewHQrXtPft2+mYSE+IIOeb6uFWl5rXZ2Mzd6T6xeuZRLly7Sq8/AHK93f7Q3ixZ+zvZtm7CxseGDKTNveUalKN+TIiL3OhU0IlLqvf7meze8XqIwUlOSAfDx+ft+BY6OTji7uHIuNcVyWhWAj285y+MyZcpQzs+P5OQkzpwp+Dfx1+6FAFiWAXZ0dMrxPD8pyWd5oldnEhNP06FjZ6Z+OMfyWtq5VAY92YMdv23Fr3xF6tWrj59fedLOpRYor8KMg4eH53XbOAKQVYgL6POzd88uILtgKqwbvScWf/tfgmrUolHjZrle69a9J9u3baJW7Xq39Z4qyvek3B4PDw+d6idyh3h4eBRJHBU0IiJFqGrVAACi/tpvaTsVF0vauVRcXd3w9PQmhiMAHIw6QGZmJmXKlOHixQucOH4Mg8FAJf/Kln2zsrLuSJ6XL19iQP9Qoo8comGjpsz74tscMxmLv1vIjt+20rP3k3w88wsMBgN9e3Xm8KGoXLHyyrEw43AnPixu3LCO//33MwC6dnusyOIePx7Dzh3bcizffL1rx6I7g987UlJSrJ2CiNyEChoRKfUG9gvNdbfnjg915b1JHxc6VvuOnaldpx6/bt/M62NeokHDpvx3wVzMZjMvDBuVo2hIOH2KZ57qzcOP/JulYd9y8eIFunTtgbOzC47/f4PGz+fPJCnpDM2at7q9g/yHdyaM4feIHQDUCw5h5sdTLK+NHD3ecp+TxMTTHIw6QNRfkfy2fQvw9+zJjXIszDgUlWu/x7TzaZxLzf4Q2qJla/r2G5zvttdbumLDDV/v+FBXPP//wvzHej5RJLkWtv9beU+KiNzrVNCISKmX1/UaZ88m3VIsOzs7vlr0I68Of4b/LpjHfxfMw87enuGvvM6Lw3LehLNO3fsAGDFsCJB9DcykaTMB6Nf/KbZu+YXfft3C1atXirygOX3672NetPDzHK+NePUN+vQdyLKwb9m88WfabWxA1aoBNGt+P9u2biQu7uRNcyzMOBSVa79HW1tbqgUE8tjjT/Diy6Mty0Tnte31TNed5pbX60lJZwhfv5o2D3a03IvodnMtTP+3+p4UEbnXGcxms9naSYhI6RB3JoPer5/EZDLzWDt3+nd2t3ZKd1RCQjznUlPxr1wFp/+fzQCIMP5K9y5taNykOSt/2sbJ2OOYgcqVq+bYPysri/TzaZR1cLDKBeFms5nY2OOUtbfHr3zFPLcpSI75jUNhXb16hWqVXLC3L0v0iXMYDIZiPbUrIyODyH17qFix0m0XNLcjMzOTbVs20LdXZ+5v1YYffgwvtr4X/3yOb9edI/2Smae7u+PrWTK/F7181cyMr8/SpZUr04b7WTsdEcnH1IVJrNhynkXv+lOtQv7XhZbM/xOJiJ
```

Comment:
typo: functionf

Reply:
i have hosted the image so i dont have to export it as a string like that. please see new commit to see if that solved

Reply:
added image as pdf, please see comment above